docs/testing/manual-testing.md

```diff
@@ -82,12 +82,12 @@ to test their appearance too:
 Here's how we're going to test the message appearances:
 
 - narrow to a new topic and send a message (this message will include sender)
-  - edit the message ("(EDITED)" label should appear beside sender name)
+  - edit the message ("EDITED" label should appear beside sender name)
 - send another message (will not include sender)
-  - edit the message ("(EDITED)" label should appear in the left column, where the avatar is)
+  - edit the message ("EDITED" label should appear in the left column, where the avatar is)
```

Comment:
You missed removing the left-parens here.  Fixed in #21046 which I happened to see first, but please self-review your work before submitting a pull request -- bugs like this can be very embarrassing for the project, and it shouldn't be my job to catch sloppiness like this.

Reply:
I'm very sorry for this mistake, will surely take care of it in future.

Reply:
umm, @timabbott , I don't think i've missed the left-paren here, I left it purposely as they are not around ```EDITED``` .

They are around the whole text ```( "EDITED" label should appear in the left column, where the avatar is)```
I guess it's wrongly done in #21046 rather than in this.

Reply:
Ahh, quite so, I misread :)